README.md

```diff
@@ -12,57 +12,69 @@ install.packages("remotes")
 remotes::install_github("https://github.com/palantir/palantir-r-sdk", ref = "0.5.0")
 ```
 
-This library relies on [palantir-python-sdk](https://github.com/palantir/palantir-python-sdk), so Python must be installed on the system.
-The following command can be used to install the Python SDK and required dependencies:
-```
-palantir::install_palantir()
-```
+### Configuration
+
+Configuration for hostname and an authentication token can be provided by environment variables (`FOUNDRY_HOSTNAME`, `FOUNDRY_TOKEN`)
+
+* `FOUNDRY_HOSTNAME` is the hostname of your instance e.g. `example.palantirfoundry.com`
+* `FOUNDRY_TOKEN` is a token acquired from the `Tokens` section of Foundry Settings 
+ 
+Alternatively, you can set the following options:
+
+* `foundry.hostname` is the hostname of your instance e.g. `example.palantirfoundry.com`
+* `foundry.token` is a token acquired from the `Tokens` section of Foundry Settings 
+* `foundry.timeout` is the timeout of any HTTPS request made to Foundry
 
-Configuration for hostname and an authentication token are provided by environment variables (`PALANTIR_HOSTNAME`, `PALANTIR_TOKEN`)
+Finally, you can create a YAML file called `~/.foundry/config` to define the hostname and token:
+
+```yaml
+hostname: example.palantirfoundry.com
+token: ...
+```
 
-* `PALANTIR_HOSTNAME` is the hostname of your instance e.g. `example.palantirfoundry.com`
-* `PALANTIR_TOKEN` is a token acquired from the `Tokens` section of Foundry Settings 
+Configuration will be loaded in the above order: environment variables if present, otherwise options, otherwise config file.
  
 Authentication tokens serve as a private password and allows a connection to Foundry data. Keep your token secret and do not share it with anyone else. Do not add a token to a source controlled or shared file.
 
+### Aliases
+
+To read datasets, aliases must first be registered. Create a YAML file called `~/.foundry/aliases` and define an alias for every dataset you wish to read or write to.
```

Comment:
Would be good to get API/Transform team opinion on this.

Reply:
This is not relevant anymore as we support internal APIs